ludic/attrs.py

```diff
@@ -472,6 +472,34 @@ class SvgAttrs(GlobalAttrs, total=False):
     xmlns: str
     height: int
     width: int
+    viewBox: str
```

Comment:
I would rename it to python style:

```suggestion
    view_box: Annotated[str, Alias("viewBox")]
```

Reply:
Also, please run pre-commit hooks, so that he pipeline passes :)